apps/playwright/fixtures/AppPage.ts

```diff
@@ -0,0 +1,86 @@
+import { type Page } from "@playwright/test";
+
+import { Test_MeQueryDocument } from "~/utils/user";
+import { FeatureFlags, getFeatureFlagConfig } from "~/utils/featureFlags";
+import { getAuthTokens } from "../utils/auth";
+
+/**
+ * App Page
+ *
+ * Common functionality, extended by other pages
+ */
+export class AppPage {
+  constructor(public readonly page: Page) {}
+
+  async gotoHome(locale: "en" | "fr" = "en") {
+    // Timeout is for Firefox having issues with this navigation
+    await this.page.goto(`/${locale}`, { timeout: 120 * 1000 });
+  }
+
+  /**
+   * GraphQL Request
+   *
+   * Make a GraphQL request using the current page context
+   *
+   * @param query
+   * @param variables
+   * @returns
+   */
+  async graphqlRequest(query: string, variables?: Record<string, unknown>) {
+    await this.gotoHome();
```

Comment:
Why is this part necessary? I would think this step would just slow down graphql requests. If its to make sure the authoirzationQuery has run, would it be possible to try to get auth tokens, and if they don't exist yet, _then_ goToHome?

Reply:
This is related to the other comment...sort of. So, our tokens were expiring before the test ended and this request was actually happening outside of the browser. So this just goes home, waits for the token to be refreshed and then sends off the request. I think we can make this better in one of three ways:

1. As mentioned in other comment, intercept refreshes in the in the fixture and keep our storage state up to date (probably best?)
2. Right a refresh to use here and in the other graphql utility to refresh if the token is expired. Basically, mimic what our client provider is doing
3. Just be lazy and extend our dev tokens to something like 30mins so this is unnecessary 😅 

If you'd like, I could see about getting one of these solutions working here before merging.